pages/_app.tsx

```diff
@@ -70,17 +70,6 @@ export default function App({ Component, pageProps }: AppProps) {
     dialogRef.current?.close();
   };
 
-  useEffect(() => {
```

Comment:
Have we decided to remove click outside to close function? Maybe let's create a ticket to add it back if we like it?

Reply:
the previous implementation stopped all event propagation in the dialog box and I was trying to get the PR ready for release last wednesday. we can make a low-priority ticket to re-add #37